### DISCUSSION TITLE
Changes for Children

### DISCUSSION THREAD

**Discussion by arbadacarbaYK** [BitcoinDesign/Guide#606] [Ideas]
I would like to discuss some features for childrens wallets that would be used regularily and not for savings so for their pocket money. Some things for them or the parents would need inclusion in sw wallets or within the node and support by design team

Lets say we use a Node as the bank for our family - so we set up multisig wallets for your children lets say fromon 6. 
Today we are missing sw wallets that fit this usertype and features to support then as well.

A few things for the kids part
- mobile wallet should be easy to understand even if you cannot read e.g. use avatars for a whitelist of adresses 
- mobile wallet should show a savings account where they can just send to/view
- pocket money wallet should have "send to savings account" 
- children lose their phone quite ofte so backups should be made on every change to a cloud-space on the node, dropbox, wherever
- own linklist-store (see below)
- do not show the funds of the child when sending money
- themes to grow with the app
- gamification as long as this is ethical and legal (gambling itself as well as winning money)

<img width="349" alt="Bildschirmfoto 2022-10-25 um 14 07 02" src="https://user-images.githubusercontent.com/63317640/197804644-66574cc5-fa87-437a-9c97-2932e234c47c.png">

a few things for the node part
- weekly/monthly/yearly Pocket-money service to multisig-wallets or
a daily/weekly/monthly spending limits intead of a separate savings account
- Blacklist (controversial, i know) for know spam adresses like e.g. https://scam-alert.io/ hosts them 
- Blacklist to be used to block bitrefill, sony playstation, tippin.me whatever (lot of in-game coins and spam coming)
- Faucets/airdrops/interest from the bank to the child if they solve random questions or small in-wallet games or if they saved up a certain amount as a  motivation to save
- child gets 18 and the wallet at some stage needs to become a non custodial wallet, maybe even without leaving behind empty adresses 
- drop a linklist to a sw-wallet to some services for kids like call-me-a-cab, call/message parent whereby call opens via voip maybe some have more or better ideas

<img width="342" alt="Bildschirmfoto 2022-10-25 um 14 07 17" src="https://user-images.githubusercontent.com/63317640/197805219-9a2c8216-8b18-46cd-85c4-d3b1f160d5ac.png">

Im interested in your thoughts and ideas on this :)

All Images are a result of the Designathon2022  and were made by David Nordh on the base of the ongoing discussion you find here
https://docs.google.com/document/d/1JFFDINZEP-g1vp4mh9JcX0IrDlkBeUYv_Q9A85T1xZ8/edit

<img width="348" alt="Bildschirmfoto 2022-10-25 um 14 06 09" src="https://user-images.githubusercontent.com/63317640/197805096-8e5b7693-676f-4c04-bf27-0be74945f6a1.png">

<img width="359" alt="Bildschirmfoto 2022-10-25 um 14 06 56" src="https://user-images.githubusercontent.com/63317640/197805167-07acd53a-8ddb-425d-9646-46f07a5e9602.png">
<img width="347" alt="Bildschirmfoto 2022-10-25 um 14 06 49" src="https://user-images.githubusercontent.com/63317640/197805176-c1489933-2352-4e1c-afb0-ae58107fc77d.png">

**Comment by GBKS** [comment]
Auch a good topic. Thanks for posting.

It might be best to start with looking at the relationship kids have with money and how it changes over time, along with how parents and kids interact around money.

For example, my son has always saved his allowance. He's a natural stacker, if you will. His cousin likes to spend money on gifts for friends and family. Other kids like to buy candy or toys for themselves, etc.

Knowing my son handles money well, I was comfortable setting him up with a self-custodial wallet at age 9. If he had not managed his money well, I would have preferred a different setup where maybe only part of the allowance was available for free spending and the rest needed approval.


If we could map out these dynamics, we could then derive a set of features, personas, a usage life cycle, how this whole experience should feel, etc.

This topic could make for a cool design concept case study.

**Reply by arbadacarbaYK** [reply]
Please let me know how and where to contribute to some discussion on this 🤗

**Reply by GBKS** [reply]
I think right here is a great place to discuss.

**Reply by arbadacarbaYK** [reply]
Happy New Year 🥳🍾

I  agree that up to a certain age most do handle their pocket money quite well. I´ll start telling how far we came discussing a blacklist as the most uncomfortable topic on the list 😅 because it might indeed be cuicial for the safe and positive UX of btc/ln for younger generations.

Children will get more and more in contact with not only bitcoin but with the "crypto" world. Soon, if not already the case, game consoles and social media platforms will offer sending/receiving/earning various "digital currencies" and children will see bitcoin as one that competes with some projects thatt have high budgets on fancy marketing, celebrety influencers, free in-wallet-games attached, themed wallets, catchy ads etc. It is likely their favorite game company or influencer might not promote bitcoin/LN but rather a sponsored product of a centralized / possibly scam unit, that is not only requesting their time and data but will maybe cost them all their funds. All "digital currencies" allow users of every age to send their funds unlimited and to every other address by design. 

Social media platforms are allowed fromon 13, pocket money should be given fromon at least 7.
Children between 7-17 are allowed to spend their pocket money on whatever they like - if the parents do not disagree and if the product is suitable for the kids age. In return one could say parents are somehow responsible for kids aged between 7-17. Most offline and online shops act accordingly for cash or money transfer and ask for a proof of age on intersection (gaming prepaid cards offline only fromon 18, online only by CC ). 
Some online platforms selling, charging or trading "digital currencies" are KYC free and thereby do not have an age-proof as well 📉

New regulatory aspects for digital money transfered by children through the most common social media platforms will be made up long after the currency has been included since they have no intrinsic interest in limits and will point this to the parents as long as they can. This can not be prevented by national regulations.

Triggers to be limited/blocked approach the child from various directions (send me 1 BTC I´ll send you 2, switch BTC here against BERT-coin for new weapons, I will send you cheat codes if you tip me,  🚩🚩..)

The topic hits the field of parental control apps*, special bank accounts for pocket money and mobile phone kids tariffs - to set limits for time on apps & money is quite a usual approach -but always guarded by a 3rd entity and tbh come with a lot of features on top, that you would rather see running on your own node instead of 3rd party data collectors servers.

Questions that came up 
Isnt´t this against the idea of bitcoin ?
Is this your job as a parent/bank ? 
Isnt it better to return to fiat cash with kids if overspending gets a problem ? 
Isn´t it a part of money education, that they lose it all - even if recurring ? 
Do the blacklists in place assure they are not misused by the author? 
How to include a blacklist - like a warning message (the receipients adress fits to a known spam adress) or more like a strict block ? 
Shouldn´t other wallets on a node with such features be warned, that the "bank" could drop a blacklist on them, too ?
Would adults want to use this for themselves as well ? 

Reasons for a whitelist were the very young ones  
It is likely, that if you cannot read or write you may enter a wrong receipients address 
Minors could handle symbols better than values
Might be easier to handle technically because it is shorter than a personal blacklist 

😅

**Comment by ShalomOhuegbe** [comment]
Hi @arbadacarbaYK this is a nice idea but won't you prefer making this app an app for kids to earn by learning about bitcoin? I feel that gamifying the app and using lots of animations will make the whole process less 'serious' what do you think?

**Reply by arbadacarbaYK** [reply]
Hey, yes, that point found its way in the designathon concept. Gamification for money is only allowed fromon a certain age so it may not be as proninent as one would think of first.
